example/timer/odp_timer_accuracy.c

```diff
@@ -0,0 +1,461 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <stdlib.h>
+
+#include <unistd.h>
+#include <getopt.h>
+
+#include <odp_api.h>
+
+#define START_OFFSET_NS ((uint64_t)(300 * ODP_TIME_MSEC_IN_NS))
+
+typedef struct test_global_t {
+	struct {
+		long int period_ns;
+		long int res_ns;
+		int num;
+		int init;
+	} opt;
+
+	odp_queue_t      queue;
+	odp_timer_pool_t timer_pool;
+	odp_pool_t       timeout_pool;
+	odp_timer_t     *timer;
+	uint64_t         period_ns;
+	uint64_t         first_ns;
+
+} test_global_t;
+
+static void print_usage(void)
+{
+	printf("\n"
+	       "Timer accuracy test application.\n"
+	       "\n"
+	       "OPTIONS:\n"
```

Comment:
optional OPTIONS

Reply:
Options are by default optional, so didn't change. Some of our applications have exceptional options that are mandatory (e.g. interface name).